x-pack/legacy/plugins/alerting/server/types.ts

```diff
@@ -76,6 +76,8 @@ export interface Alert {
   scheduledTaskId?: string;
   createdBy: string | null;
   updatedBy: string | null;
+  createdAt: Date;
+  updatedAt: Date | null;
```

Comment:
hmmm ... now I'm really wondering about when `updatedAt` can be null, and `createdAt` can't be null.  That doesn't seem right.  Upon creation, I'd think `createdAt` and `updatedAt` would be the same value, and then `updatedAt` will get updated from then on.  So, `updatedAt` can't really ever be null, seems like.